compose-dev.yaml

```diff
@@ -3,3 +3,17 @@ services:
     build: .
     ports:
       - "8000:80"
+  postgres:
+    image: postgres:16-alpine3.20
+    shm_size: 128mb
+    env_file: ".env"
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
```

Comment:
Also add these environment flags to the backend container to remove the need for copying the `.env` file